components/dropdown/dropdown-menu.js

```diff
@@ -22,11 +22,20 @@ class DropdownMenu extends ThemeMixin(DropdownContentMixin(LitElement)) {
 		super();
 		this.noAutoFocus = true;
 		this.noPadding = true;
+		this._menuFocused = false;
+		this._maxHeightNonTray = this.maxHeight;
 	}
 
 	firstUpdated(changedProperties) {
 		super.firstUpdated(changedProperties);
 
+		this._maxHeightNonTray = this.maxHeight;
```

Comment:
I decided to ignore the max-height for all menus in trays. For cases like the user profile card, we do want to set that maximum height in a tray (since the card is quite "short" and would look odd with the extra space), so I decided to keep the max-height in the dropdown mixin.